### ISSUE TITLE
Hexapod driver initial framework

### ISSUE THREAD

**Issue by iainworkman** [acquaman/acquaman#1460]
The hexapod driver needs a command line structure setup for command to be run, and for queries to be made to a controller. This will require a console application created which will provide an application user interface for interacting with the controller through the driver.

**Comment by dretrex** [comment]
Time Estimate: 6 hours

**Comment by dretrex** [comment]
Copied by @dretrex
There are a few notes about the application in its current (7da0c24) state:

I had to remove mobility from the config file in acquamanCommon.pri as it seemed to be causing issues. Will look at returning this as we move closer to deploy.
In order for this to build the PI_GCS2 drivers need to be located in ~/beamline/programming/acquaman/contrib/pi_gcs2/lib. Again, closer to deploy this will be brought closer to the standard AM way of including libraries. If anyone needs a copy of the .so file, let me know.
The build requires glibc v2.7 which is not available on SL5.4. We can get access to a virtual opi running SL7.1 for building. For those not using SL5.4 this shouldn't be an issue.
At the moment the AMConsoleApplication stuff I wrote is bundled in the the new AMPIC887DriverServer project. This can (and should) be pulled out in its own project in the fullness of time.
I had to edit the included .h file for the PI GCS2 library. It was missing an include.
